### ISSUE TITLE
A problem about training

### ISSUE THREAD

**Issue by sunsun-rgb** [alon-albalak/D-REX#1]
Hi,Alon
 After having trained the full system, I noticed that the best f1 score of the  intitial ranking model when pretraining  is higher than that of the full system and much higher than  BERTs which it follows,  that makes me confused , could you tell me the reason?

**Comment by alon-albalak** [comment]
That is strange. Can you provide the scores you got for each method on the dev/test sets?

Also, if you didn't train the model with `bash train_drex_system.sh $GPU` then can you briefly explain how you trained those models?

**Comment by sunsun-rgb** [comment]
I trian it with bash train_drex_system.sh $GPU , here is the results on dev set of initial ranking model when first pretraining it[train_outputs.log](https://github.com/alon-albalak/D-REX/files/8392882/train_outputs.log) and this is the  result of the full system 
[train_outputs.log](https://github.com/alon-albalak/D-REX/files/8392942/train_outputs.log).

**Comment by alon-albalak** [comment]
It looks like your results are different from ours: [initial_ranking_model_outputs.log](https://github.com/alon-albalak/D-REX/files/8400703/initial_ranking_model_outputs.log) and [D_REX_system_outputs.log](https://github.com/alon-albalak/D-REX/files/8400706/D_REX_system_outputs.log)

The policy gradient that we use could use better stability, and there are some possible causes for worse performance on the full system (this can definitely be improved):
- Different python package versions from those specified in the requirements.txt file. I believe there are some changes in pytorch that occured in future versions that affect the system's learning
- Using CUDA 11.x has some changes as well. Our experiments were run on CUDA 10.2
